bundles/org.openhab.binding.homematic/README.md

```diff
@@ -743,3 +743,9 @@ stop org.openhab.binding.homematic
 log:set TRACE org.openhab.binding.homematic
 start org.openhab.binding.homematic
 ```
+
+### Running in Docker
+
+First of all you need map the `XML-RPC` and  `BIN-RPC` Callback Ports to the outside world.
```

Comment:
"need to map"?

Reply:
Ah yes. I reorganized those two sentences too often.  I originally wanted them in the inverted order, but I just didn't get it to work out. :-)